lessons/beginners/list/index.md

```diff
@@ -354,25 +389,34 @@ for cislo in range(10):
 print(mocniny_dvou)
 ```
 
+Podobným způsobem získáš seznam seznam `matka`, `babička`, `prababička`,
+`praprababička`, atd.:
+
+```python
+predkove = ['matka']
+for pocet_pra in range(10):
+    predkove.append('pra' * pocet_pra + 'babička')
+print(predkove)
+```
+
 Chceš-li seznam, který reprezentuje balíček karet,
-zavolej `append` pro všechny kombinace barev a hodnot.
+zavolej `append` pro všechny kombinace barev a hodnot:
 
 ```python
 balicek = []
-for barva in '♠', '♥', '♦', '♣':  # (Na Windows použij textová jména)
+for barva in '♠', '♥', '♦', '♣':
```

Comment:
Nebude matoucí, že je toto pokoutná čtveřice? Když už bereme ty seznamy, nemůže to být seznam?

```suggestion
for barva in ['♠', '♥', '♦', '♣']:
```

Reply:
Nemyslím si:  <var>n</var>-tice se takhle používají už v lekci o cyklech. Jen není řečeno jže jde o <var>n</var>-tici.